apps/docs/src/components/mobile-nav.tsx

```diff
@@ -23,7 +23,7 @@ export function MobileNav() {
           <span class="sr-only">Toggle Menu</span>
         </As>
       </SheetTrigger>
-      <SheetContent size="xl" position="left" class="pr-0">
+      <SheetContent position="left" class="pr-0">
         <MobileLink href="/" onOpenChange={setOpen} class="flex items-center">
```

Comment:
Should these just use an anchor tag with an `onClick` instead of `<MobileLink />`?